docs/content/error/$http/baddata.ngdoc

```diff
@@ -0,0 +1,16 @@
+@ngdoc error
+@name $http:baddata
+@fullName Bad JSON Data
+@description
+
+The default @{link ng$http#default-transformations `transformResponse`} will try to parse the
+response as JSON if the `Content-Type` header is `application/json` or the response looks like a
+valid JSON-stringified object or array.
+This error occurs when that data is not a valid JSON object.
+
+The error message should provide additional context such as the actual response.
+
+To resolve this error, make sure you pass a valid JSON data object to `transformResponse`.
+
+For more information, see the {@link ng$http#default-transformations `transformResponse`} service
```

Comment:
This is not necessary, since we already link to `default-transformations` above imo.